src/main/java/com/mraof/minestuck/world/biome/LandBiomeHolder.java

```diff
@@ -128,15 +128,11 @@ private static void addDefaultOres(BiomeGenerationSettings.Builder builder, Stru
 	{
 		if(MinestuckConfig.SERVER.generateCruxiteOre.get())
 		{
-			builder.addFeature(GenerationStep.Decoration.UNDERGROUND_ORES, PlacementUtils.inlinePlaced(Feature.ORE,
-					new OreConfiguration(blocks.getGroundType(), blocks.getBlockState("cruxite_ore"), baseCruxiteVeinSize),
-					CountPlacement.of(cruxiteVeinsPerChunk), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(cruxiteStratumMax)), BiomeFilter.biome()));
+			builder.addFeature(GenerationStep.Decoration.UNDERGROUND_ORES, MSPlacedFeatures.CRUXITE_ORE.getHolder().orElseThrow());
```

Comment:
Lands shouldn't use these ore features that are defined for the overworld. They should be using the ore block and ground type provided by the `StructureBlockRegistry`. Please revert the changes to this class.